linkerd/docs/client_tls.md

```diff
@@ -66,10 +66,11 @@ protocols         | unspecified                                | The list of TLS
 
 If present, a client auth object must contain two properties:
 
-Key      | Default Value | Description
----------|---------------|-------------
-certPath | _required_    | File path to the TLS certificate file.
-keyPath  | _required_    | File path to the TLS key file.  Must be in PKCS#8 format.
+Key               | Default Value | Description
+------------------|---------------|-------------
+certPath          | _required_    | File path to the TLS certificate file.
+clientCaChainPath | none          | Path to a file containing a CA certificate chain to support the client certificate.
```

Comment:
Why does just this setting have the word "client" in its name and in its description, but the other settings don't? It seems like "client" is redundant here and could/should be removed.

Reply:
This is to mirror the `serverCaChainPath`.  Of course, that might be a poor name also.  My intention was to try and avoid confusion between the CA that the server uses to validate the client identity (`caCertPath` in the server tls struct) and the CA that issued the server's cert (`serverCaChainPath` in the server tls struct).  

Likewise I wanted to differentiate between the CA that the client uses to validate the server identity (`trustCerts` in the client tls struct) and the CA that issued the client's cert (`clientCaChainPath` in the client auth section of the client tls struct).

Any suggestions for better names for `clientCaChainPath` and `serverCaChainPath` would be appreciated.  The names of any other properties are more difficult to change because they have already been released and require a breaking config change to rename.

Reply:
I don't have strong feelings about this. Since both already have "keyPath" and "certPath" it makes sense to me that both would use the same name for this third thing too. Perhaps "intermediateCertsPath" might be a reasonable name, since that implies that (1) these aren't the trusted roots, and (2) the roots shouldn't be included in the chain.

Reply:
Great suggestion, thanks!